src/components/ButtonDetails/ButtonDetails.jsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Typography } from "@mui/material";
+import { buttonDetailStyle } from "../../buttonDelivery.styles";
+import { BsLayoutTextSidebar } from "react-icons/bs";
+
+const ButtonDetails = () => {
+  return (
+    <Button
+      sx={buttonDetailStyle}
+      endIcon={<BsLayoutTextSidebar style={{ color: "#00000075" }} />}
+    >
+      <Typography
+        sx={{
+          color: "#000",
+          fontWeight: "500",
+          fontFamily: "'Inter', sans-serif",
+          fontSize: 16,
+        }}
+      >
+        Details
+      </Typography>
+    </Button>
+  );
+};
```

Comment:
Please remember that you don't need to put a <Typography /> component inside a button, that's an antipattern. You should only have text

Reply:
ok well.